### ISSUE TITLE
require --dev security-csrf and form for this package?

### ISSUE THREAD

**Issue by dr-matt-smith** [sadikoff/crud-maker-bundle#10]
since won't work without **security-csrf** and **form**
should these be added to compoer require --dev for this package?

.. matt ..

**Comment by sadikoff** [comment]
I don't think that we need to add this. we need somehow to detect if crsrf is enabled and generate code according to this detection/

**Comment by sadikoff** [comment]
I added CSRF dependency to command, by default symfony has enabled session and csrf and after some discussions I decided that it will be the best solution. 

new release is ready, I think everything works, but some tests with feedback will be very useful =)
